promscale/visualize-data/jaeger.md

```diff
@@ -0,0 +1,38 @@
+# Connect Promscale and Jaeger Query
+This section shows you how to connect Promscale with
+[Jaeger][grafana-jaeger-datasource].
+
+Before you begin, you need the IP address of your Promscale instance. You can
+find this at the command prompt, using this command:
+```bash
+docker inspect <PROMSCALE_CONTAINER_NAME>
+```
+
+The IP address is listed in the `NetworkSettings → Networks → IPAddress`
+section.
+
+Alternatively, you can set the URL as `promscale:9202` (here `9202` is the gRPC endpoint of Promscale), where `promscale` is the name of the container.
+
+<procedure>
+
+### Connecting Promscale and Jaeger Query
+1.  Install Jaeger from the [official Docker image][jaeger-docker]:
+    ``` bash
+    docker run -d \
+      -p 16686:16686 \
+      --network opentelemetry-demo_default \
+      --name=jaeger \
+      -e "SPAN_STORAGE_TYPE=grpc-plugin" -e "GRPC_STORAGE_SERVER=promscale:9202" \
+      jaegertracing/jaeger-query:1.30
+    ```
+1.  Navigate to `localhost:16686` in your browser, you will able to access the Jaeger query homepage.
```

Comment:
This is only try if you install Jaeger on your local machine which nobody will do.